examples/dog-mode/AppManifest.json

```diff
@@ -31,7 +31,7 @@
         {
             "type": "grpc-interface",
             "config": {
-                "src": "https://github.com/eclipse/kuksa.val.services/blob/v0.2.0/hvac_service/proto/sdv/edge/comfort/hvac/v1/hvac.proto",
```

Comment:
don't we have the hvac in the new Eclipse Kuksa org?

Reply:
Removed as not use by example code